Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -145,6 +158,19 @@ HRESULT InitializeGeometries() {
 
         return S_OK;
     }
+
+    HRESULT SimplifyGeometryToPathWithTransformation(ComPtr<ID2D1Geometry> geometry, const CGAffineTransform* transform) {
```

Comment:
>SimplifyGeometryToPathWithTransformation [](start = 12, length = 40)

I suggest renaming this function and hide the implementation details and instead call out what it does to the path.  In this case, you are adding a gemetry to path using transform, so it should be AddGeometryToPathWithTransformation().  The fact that you are using D2D1Geometry.simplify is merely an implementation detail.